docs/maintainers/generic-how-to-implement-a-concept-exercise.md

```diff
@@ -102,7 +103,7 @@ This file contains information on the exercise's design, which includes things l
 
 ## Step 11: add .meta/config.json:
 
-This file contains meta information on the exercise, which currently only includes the exercise's contributors ([example][meta-config-json]).
+This file contains meta information on the exercise, which currently includes the exercise's contributors, and language version requirements ([example][meta-config-json]).
```

Comment:
This is missing the optionally bit (same for some of the other files):

```suggestion
This file contains meta information on the exercise, which currently includes the exercise's contributors, and (optionally) language version requirements ([example][meta-config-json]).
```

Reply:
Should we perhaps remove it from the language-specific files altogether and let maintainers update it themselves? Having `(optionally)` there doesn't really tell you anything if you're looking at how to implement an exercise.

Reply:
That's a great idea!